src/client/components/Forms/SignUp.js

```diff
@@ -1,22 +1,90 @@
 import React, { useState } from 'react';
 import PropTypes from 'prop-types';
+import iconGithub from '../../assets/images/icons/github.svg';
+import './Signup.style.css';
+import Button from '../Button/Button';
 
 export default function SignUp({ onSubmit }) {
   const [email, setEmail] = useState('');
+  const [fullName, setFullName] = useState('');
+  const [position, setPosition] = useState('');
+  const [profileImageUrl, setProfileImageUrl] = useState('');
+  const [linkedin, setLinkedin] = useState('');
+  const [github, setGithub] = useState('');
+  const [website, setWebsite] = useState('');
   const [password, setPassword] = useState('');
   const [passwordConfirm, setPasswordConfirm] = useState('');
+  const [validation, setValidation] = useState('');
+
   const handleEmailInput = (e) => setEmail(e.target.value);
   const handlePasswordInput = (e) => setPassword(e.target.value);
   const handlePasswordConfirmInput = (e) => setPasswordConfirm(e.target.value);
+  const handleFullNameInput = (e) => setFullName(e.target.value);
+  const handlePositionInput = (e) => setPosition(e.target.value);
+  const handleProfileImageUrlInput = (e) => setProfileImageUrl(e.target.value);
+  const handleLinkedinInput = (e) => setLinkedin(e.target.value);
+  const handleGithubInput = (e) => setGithub(e.target.value);
+  const handleWebsiteInput = (e) => setWebsite(e.target.value);
   const handleSubmit = (e) => {
     e.preventDefault();
-    onSubmit({ email, password, passwordConfirm, fullName: 'Jane Doe' });
```

Comment:
Don't hard code any values, why `fullName: 'Jane Doe' ` ?

Reply:
thats the problem ember she used for testing because she needs this component and her branch is mered alreday in develop

Reply:
I needed default value for fullName so that I could try and post the user with stored firebase token from firebase authentication of user by email and password. However I did communicate to Kamal very clearly in previous user form PR he just closed and on several calls during our sunday classes that he does not need default value for fullName when he will be merging his PR. I also gave Kamal a heads up there would be a merge conflict and that he just needs to accept his changes and discard mine (discard the hard coded fullName value)